data/blacklist.d/porn_reddit.hosts

```diff
@@ -5856,3 +5856,46 @@
 127.0.0.1 www.assangels.com
 127.0.0.1 counter5.sextracker.com
 127.0.0.1 www.ibannerx.com
+127.0.0.1 phimmoi.net
+127.0.0.1 phimbathu.org
+127.0.0.1 topphimmoi.net
+127.0.0.1 gatherproxy.com
+127.0.0.1 spys.one
+127.0.0.1 free-proxy-list.net
+127.0.0.1 xemphimplus.net
+127.0.0.1 vietpn.com
+127.0.0.1 xemvtv.vn
+127.0.0.1 m.xemvtv.vn
+127.0.0.1 thapden.com
+127.0.0.1 vtv16.info
+127.0.0.1 zingtv.net
+127.0.0.1 dongphim.net
+127.0.0.1 phim4400.tv
+127.0.0.1 vuviphimmoi.com
+127.0.0.1 phimdayne.com
+127.0.0.1 topphimhd.com
+127.0.0.1 motphim.co
+127.0.0.1 thegioitiepthi.vn
+127.0.0.1 thhp.vn
+127.0.0.1 phimhdonline.tv
+127.0.0.1 xemphimso.tv
+127.0.0.1 bongngo.tv
+127.0.0.1 huphim.com
+127.0.0.1 didibkk.com
+127.0.0.1 phim.didibkk.com
+127.0.0.1 vietsubtv.org
+127.0.0.1 m.vietsubtv.org
+127.0.0.1 goldphim.com
+127.0.0.1 phimhdmoi.com
+127.0.0.1 vuiphim.net
+127.0.0.1 vphim.net
+127.0.0.1 24hphim.com
+127.0.0.1 phunuvietnam.vn
+127.0.0.1 thegioiblu-ray.com
+127.0.0.1 tintaynguyen.com
+127.0.0.1 xemtivi.tv
+127.0.0.1 moveek.com
+127.0.0.1 tapmoi.net
```

Comment:
This looks like movie sites.They are not porn sites. I don't think we should block these.

Reply:
Its, ok.